### ISSUE TITLE
req [nolibs version]

### ISSUE THREAD

**Issue by shikulja** [Geigerkind/DPSMate#245]
dpsmate uses dirty versions of babblelibs for different languages, so do other libraries (GraphLib)
This mess is pointless when there are multi-language versions of bubblelibs.
It is also a big additional load due to the double set of libraries.

**Comment by shikulja** [comment]
It takes a lot of work. What can be done first of all:
- NPCDB move from language modules to one in the libs\ folder
- update GraphLib\dewdrop, need some rework code for compatibility
-  Babble-Boss\Spell replace with universal versions, and move them to the libs\ folder from language modules, Babble-Spell has some inappropriate code, take it out separately
- DPSMate_Parser & loc**.lua  
- move all languages ​​to the Localisation\ directory, which will have a common entry in toc. (Regardless of the quality of translation of all languages, they are, and there are no alternatives, they will still be updated along with the core, and only the necessary lua languages ​​will be selected from the current

**Comment by Geigerkind** [comment]
Did your PR solve this?

**Comment by shikulja** [comment]
@Geigerkind 
No, I only updated the Russian localization. This does not change the structure of the add-on to be more compatible with external libraries, and a more cohesive look of files.
